### DISCUSSION TITLE
guide/layout-slots/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#230] [docs-comment]
# guide/layout-slots/

概述 主题通过 <Layout /> 和 <NotFound /> 提供了 丰富的 布局插槽，可以通过这些插槽，在 页面 的不同位置注入内容。 以便用户可以个性化的使用主题。 使用 以 <Layout /> 为例，首先，需要创建一个 客户端配置文件： .vuepress/client.ts: 然后，创建一个 .vuepress/layouts/Layo...

https://theme-plume.vuejs.press/guide/layout-slots/

**Comment by MrHaoH** [comment]
我照上面的配置了一下，为啥报这个错？
Uncaught ReferenceError: __COMMENT_OPTIONS__ is not defined      at comment.js?v=56a97645:3:24

**Reply by pengzhanbo** [reply]
将版本更新到 `rc.104` 即可，该问题已在 `rc.104` 中修复。

**Reply by MrHaoH** [reply]
npm install vuepress-theme-plume@1.0.0-rc.104，这个vuepress-theme-plume插件是吧

**Reply by MrHaoH** [reply]
好了，可以了，感谢感谢
